libs/dh/core/shell/src/lib/dh-core-shell.component.ts

```diff
@@ -53,11 +54,13 @@ import { DhPrimaryNavigationComponent } from './dh-primary-navigation.component'
 })
 export class DhCoreShellComponent {
   private readonly dhTopBarStore = inject(DhTopBarStore);
+  private releaseToggleService = inject(DhReleaseToggleService);
   breadcrumbService = inject(DhBreadcrumbService);
 
   titleTranslationKey = toSignal(this.dhTopBarStore.titleTranslationKey$);
 
   constructor(inactivityDetection: DhInactivityDetectionService) {
     inactivityDetection.trackInactivity();
+    this.releaseToggleService.refetch();
```

Comment:
question: does this constructor ever run multiple times? if not, is the refetch necessary?